paint/util/__init__.py

```diff
@@ -1,3 +1,10 @@
+import logging
+import sys
+from pathlib import Path
+from typing import Union
+
+import colorlog
+
 from . import paint_mappings as mappings
 from .gauss_kruger_converter import convert_gk_to_lat_long
 from .preprocessing import (
```

Comment:
Do we want absolute or relative imports?

Reply:
I know for `ARTIST` we had problems when using absolute imports in the ``__init__.py`` files. Therefore, we decided to use relative imports for the ``inits`` and absolute imports everywhere else...